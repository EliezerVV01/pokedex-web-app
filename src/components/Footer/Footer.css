footer{
    background:#fff;
    display: block;
    text-align: center;
    color: rgb(94, 94, 94);
    box-shadow: 1px 1px 5px rgba(0,0,0,.3);
    font-weight: 600;
    padding: 10px;
    box-sizing:border-box;
    width: 100%;
    position: fixed;
    left: 0;
    bottom: 0;
}