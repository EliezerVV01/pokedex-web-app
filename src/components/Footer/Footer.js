import React from 'react';
import './Footer.css'

const footer = ( props ) => (<footer>Author: Eliezer Veras Vargas, 2019 Â©</footer>);

export default footer;