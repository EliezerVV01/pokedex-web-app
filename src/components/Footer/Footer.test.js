import React from 'react';
import Footer from './Footer';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<Footer></Footer>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});