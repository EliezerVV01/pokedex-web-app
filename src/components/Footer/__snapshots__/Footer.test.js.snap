// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer>
  Author: Eliezer Veras Vargas, 2019 Â©
</footer>
`;
